mypy.ini

```diff
@@ -17,3 +17,6 @@ ignore_missing_imports = True
 
 [mypy-scipy.*]
 ignore_missing_imports = True
+
+[mypy-pymc_experimental.*]
+ignore_missing_imports = True
```

Comment:
Why?

Reply:
without it it was throwing errors about missing stubs (which as I know aren't available for pymc-experimental). I also opened before the issue in this repo about linters and checks in pymc-experimental being different than pymc-experimental to the point that what passes linters in one crashes the linters in the other, but it didn't get much response